apps/enrolla-server/src/ee/integrations/organization-managers/impl/propel-auth.organization-manager.ts

```diff
@@ -1,23 +1,42 @@
 import { ConfigurationsService } from '../../../../configurations/configurations.service';
 import { Organization } from '../entities/organization.entity';
-import { OrganizationManager } from '../organization-manager.interface';
-import { OrganizationCreateInput } from '../dto/organization-create.input';
+import { OrganizationManager } from '../../../../organizations/organization-managers/organization-manager.interface';
+import { OrganizationCreateInput } from '../../../../organizations/organization-managers/dto/organization-create.input';
 import { HttpService } from '@nestjs/axios';
 import { firstValueFrom } from 'rxjs';
+import { Integration } from '../../integration.interface';
 
-export class PropelAuthOrganizationManager implements OrganizationManager {
-  static readonly PROPEL_AUTH_API_KEY_CONGIURATION_KEY = 'propel_auth_api_key';
-  static readonly PROPEL_AUTH_DOMAIN_CONGIURATION_KEY = 'propel_auth_domain';
+export class PropelAuthOrganizationManager
+  implements OrganizationManager, Integration
+{
+  static readonly PROPEL_AUTH_ENABLED_CONFIGURATION_KEY = 'propel_auth_enabled';
+  static readonly PROPEL_AUTH_API_KEY_CONFIGURATION_KEY = 'propel_auth_api_key';
+  static readonly PROPEL_AUTH_DOMAIN_CONFIGURATION_KEY = 'propel_auth_domain';
 
   constructor(
     private configurationsService: ConfigurationsService,
     private httpService: HttpService
   ) {}
 
+  async isEnabled(tenantId: string): Promise<boolean> {
+    return await this.configurationsService.getValue(
+      tenantId,
+      PropelAuthOrganizationManager.PROPEL_AUTH_ENABLED_CONFIGURATION_KEY
+    );
+  }
```

Comment:
can we have more than one org manager configured at a time?
If not, this can be a single key with different values.

Reply:
Let's do it in a separate PR, cause we'll need to change stuff in staging/prod configs